CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.0.0 (2017-02-15)
+
+* Added support for pk7s (need to pass gen=7 option to parser)
```

Comment:
You're right, never mind. (Now that I think about it we might also want to document `pkparse.SUPPORTED_GENS`, but that's probably less important.)